libs/dal/src/repositories/preferences/preferences.schema.ts

```diff
@@ -32,42 +32,35 @@ const preferencesSchema = new Schema<PreferencesDBModel>(
       all: {
```

Comment:
As above in `UpsertPreferences`, we only store the minimal provided preferences now to support `PATCH` operations. Therefore it's necessary to remove all defaults from the DAL.